### PR TITLE
Add recipe for VASE by David A. Parry

### PR DESCRIPTION
**Pull request by mptrsen**
Add recipe for VASE

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mptrsen** [comment]
@BiocondaBot please add label

**Comment by mptrsen** [comment]
@BiocondaBot please add label

**Comment by mptrsen** [comment]
@BiocondaBot please update

**Comment by mptrsen** [comment]
@BiocondaBot please add label

**Comment by mptrsen** [comment]
@bioconda/rpetit3 please approve. With the merge of #37174 all dependencies for both versions of VASE are in Bioconda and this recipe also passes all tests.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
Do we need the old 0.2.4 version?

File: recipes/vase/0.2.4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - vase -h
+
+about:
+  home: https://github.com/david-a-parry/vase
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Variant Annotation, Segregation and Exclusion for family or cohort based rare-disease sequencing studies"
+  doc_url: 
+  dev_url:
```
Comment (dpryan79) [inline recipes/vase/0.2.4/meta.yaml:47]: Please delete the empty doc_url and dev_url entries.

**Comment by mptrsen** [comment]
> Do we need the old 0.2.4 version?

@dpryan79 Yes, a collaborator of mine uses a reference annotation based on 0.2.4. I don't know yet if we can replace it with 0.5.1 (the recent version).

All fixed, please approve.

**Comment by dpryan79** [comment]
The 0.2.4 packages don't go away when we update the recipe, we just stop building them. I'll approve this for now since you need it but ideally we'd later remove the 0.2.4 recipe to have less extra work during migrations.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**